homeassistant/components/blueprint/errors.py

```diff
@@ -78,3 +78,7 @@ def __init__(
             blueprint_name,
             f"Missing placeholder {', '.join(sorted(placeholder_names))}",
         )
+
+
+class FileAlreadyExists(HomeAssistantError):
+    """Error when file already exists."""
```

Comment:
You should extend the blueprint base error.

```suggestion
class FileAlreadyExists(BlueprintWithNameException):
    """Error when file already exists."""

    def __init__(self, domain: str, blueprint_name: str) -> None:
        """Initialize blueprint exception."""
        super().__init__(domain, blueprint_name, "Blueprint already exists")
```